Dockerfile

```diff
@@ -25,23 +25,33 @@
 #     -i -t --rm \
 #     --runtime=nvidia \
 #     core3d/danesfield \
-#     danesfield/tools/material_classifier.py --cuda ...
+#     danesfield/tools/material_classifier.py --cuda ...    
```

Comment:
minor nit pick, this change is adding trailing whitespace.  We usually try to avoid trailing whitespace.